user-docs/latest/getting-started/introduction.md

```diff
@@ -1,3 +1,68 @@
+This user guide contains the functional documentation related to all features available in the admin panel of your Strapi application. 
+
+![Homepage of the Admin Panel](../assets/getting-started/admin-panel-homepage.png)
+
+Before going any further into this user guide, we recommend you to acknowledge the main concepts below. They will help you to understand how Strapi works, and ensure a smooth Strapi experience.
+
+- **Development environment or Production** <br> When starting working on your application, it is in development environment, which is the status for the structuration and configuration phrase of your application and its content. After deploying your application, it is in production. This status change impacts how you can use your Strapi application, as some features are only available in development environment, such as the Content-Types Builder. In the user guide, the availability or not of a feature, depending on the application status, is always mentioned in the feature's introduction.
+
+- **Versions** <br> Strapi is constantly evolving and growing. This implies that new releases are quite frequent, to improve what is already available but also to add new features to Strapi. For every new Strapi version, we communicate through our main channels and by sending notifications both on your terminal (when launching your Strapi application), and on your application's admin panel. We always recommend to use the latest version, especially since our documentation, including this user guide, are only aligned with the latest version. If you are on an older version of Strapi, some information in this documentation may not apply to your application. 
+
+- **Roles and Permissions** <br> Some features of the admin panel, as well as the content created with Strapi itself, are ruled by a system of permissions. From your Strapi application, you have the possibility to define, at a detailed level, the roles and permissions of all admin panel users and end-users. In this user guide, all features and possible options are documented. It is however possible, depending on your role and permissions, that you may not be able to access all these features and options. In that case, please refer to the main Super Admin of your Strapi application.
+
+With all this in mind, you should be ready to start your Strapi experience!
+
+
+## Accessing the admin panel
+
+The admin panel is the back office of your Strapi application. From the admin panel, you will be able to manage content types, and write their actual content. It is also from the admin panel that you will manage users, both admin panel users and end-users of your Strapi application.
+
+::: tip NOTE
+Strapi can either be used locally, or hosted on a server, making it available at a specific URL. No matter how you choose to use Strapi, your application will work the same way. The main difference is that, if using it locally, you will need to use your terminal to launch Strapi.
```

Comment:
I would remove "The main difference is that," cause if you deploy Strapi, you will also have to start your application through the terminal and restart the application after a new deploy/update.
I agree that the application will be started all the time but from my point of view, it's confusing to say there is a difference here. I could understand that I maybe don't need to start the app on the server for production (even if it doesn't make sense to not start your app in your production server)

Reply:
Yeah I think removing that last sentence or just change it to highlight the lack of CTB access